src/xenia/emulator.cc

```diff
@@ -226,6 +226,11 @@ X_STATUS Emulator::Setup(
     backend.reset(new xe::cpu::backend::NullBackend());
   }
 
+#if XE_PLATFORM_WIN32
```

Comment:
This should be different PR. You can and should put it in: ``int WINAPI wWinMain`` in file: windowed_app_main_win.cc
I would also allow user to select priority class in config.

My only concern is fact that games internally use SetThreadPriority and they can even check it and that might interfere with games stability